features/makepot.feature

```diff
@@ -595,3 +595,150 @@ Feature: Generate a POT file of a WordPress plugin
       """
       I am being ignored
       """
+
+  Scenario: Skips excluded subfolders
```

Comment:
For this scenario, I was thinking more along the lines of adding an exclude that contains a subfolder, like this:
```
wp i18n make-pot foo-plugin foo-plugin.pot --exclude=some/sub/folder
```

I don't think testing the other way around (omitting a subfolder that was part of parent folder being excluded) is needed, as all files within a parent folder would have the same problem then.